package.json

```diff
@@ -117,6 +117,7 @@
     "@angular/platform-browser": "^10.1.0",
     "@angular/platform-browser-dynamic": "^10.1.0",
     "@angular/platform-server": "^10.1.0",
+    "@angular/pwa": "^0.1001.0",
```

Comment:
It seems unrelated to the changes

Reply:
We discussed it offline - it's related to https://github.com/SAP/spartacus/pull/10003#discussion_r531575698